src/platform/mock-local-home-platform.ts

```diff
@@ -1,101 +1,32 @@
 /*
  * Mock Local Home Platform class
- * - Interacts with bundled HomeApp
- * - Accepts scan config
- * - Interacts with virtual network
  */
 
 /// <reference types="@google/local-home-sdk" />
-import {MockNetwork, MockUDPListener, RemoteAddressInfo} from './mock-radio';
-import {DeviceManagerStub} from './device-manager';
 import {AppStub} from './smart-home-app';
+import {DeviceManagerStub} from './device-manager';
```

Comment:
Seems that only bit of functionality left in the `MockPlatform` is the intent dispatching, would it make sense to fold it into `AppStub`?

Reply:
It's worth considering, but I'm not sure I want to combine them at this point.  (I also think it paints a component diagram of the system being mocked, i.e. a developer reading the code conceptualizes the idea of the developer's app talking to a "platform" layer.)